seqr/utils/search/utils.py

```diff
@@ -157,9 +172,9 @@ def _get_variants_for_variant_ids(families, variant_ids, user, user_email=None,
         }
     dataset_type = _variant_ids_dataset_type(parsed_variant_ids.values())
 
-    return backend_specific_call(get_es_variants_for_variant_ids, get_hail_variants_for_variant_ids)(
+    return backend_specific_call(get_es_variants_for_variant_ids, get_hail_variants_for_variant_ids, _raise_clickhouse_not_implemented)(
```

Comment:
thsi function is only used for reloading saved variant json which is not needed in the clickhouse backend, so no point in implementing it